lib/rubocop/cop/rspec/multiple_subjects.rb

```diff
@@ -53,7 +49,7 @@ def on_block(node)
         def autocorrect(node)
           return unless node.method_name.equal?(:subject) # Ignore `subject!`
 
-          if named_subject?(node)
+          if node.send_node.arguments?
```

Comment:
I find it a bit unclear exactly what `node.send_node.arguments?` does, so perhaps still wrap this line as a private `#named_subject?` method?